components/concept/storage/src/main/java/mozilla/components/concept/storage/CreditCardsAddressesStorage.kt

```diff
@@ -214,6 +215,44 @@ data class CreditCard(
     }
 }
 
+/**
+ * Credit card autofill entry.
+ *
+ * This contains the data needed to handle autofill but not the data related to the DB record.
+ *
+ * @property guid The unique identifier for this credit card.
```

Comment:
`guid` will be missing, say, if we're in `onCreditCardSave`, right? E.g. we got the entry object directly from GV after user entered it into a website form?

Reply:
When we autofill a credit card, GV receives the CreditCardEntry along with the guid stored in AS. We will get back the guid if we updated the autofilled credit card in https://searchfox.org/mozilla-central/source/toolkit/components/formautofill/FormAutofillParent.jsm#719.

Reply:
What if it's a brand new card? E.g. not an autofilled `entry`, but just manually typed-in by the user?

Reply:
This should still function as a wrapper of the GV CreditCard object that gets sent back and forth between AC <-> GV depending on whether or not a credit card has been autofilled or it's a new credit card submitted from GV to AC. I am a bit fuzzy on the details, but ultimately, we should still get a prompt to see if we should save the credit card. However, if we have previously autofilled this credit card and changed a field, we will have the `guid` to use  in our Validation delegate to determine if we should be updating.